### PR TITLE
bitcoind v19 Update mempool RPCs and tests

### PR DESCRIPTION
**Pull request by benthecarman**
This should fix all the test issues so it is compatible with master

### CODE REVIEW

File: bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/BitcoindRpcClient.scala [hunk 1]
```diff
@@ -107,7 +107,7 @@ object BitcoindVersion {
   }
 
   case object Experimental extends BitcoindVersion {
-    override def toString: String = "v0.19.99"
+    override def toString: String = "v0.18.99"
```
Comment (benthecarman) [inline bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/BitcoindRpcClient.scala:110]: I shouldn't have changed this in my first PR

File: bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/RawTransactionRpc.scala [hunk 2]
```diff
@@ -90,10 +91,18 @@ trait RawTransactionRpc { self: Client =>
 
   def sendRawTransaction(
```
Comment (nkohen) [inline bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/RawTransactionRpc.scala:95]: add a scaladoc explaining the semantics of setting `maxfeerate = 0`

File: bitcoind-rpc/src/main/scala/org/bitcoins/rpc/jsonmodels/BlockchainResult.scala [hunk 3]
```diff
@@ -159,22 +175,72 @@ case class GetMemPoolResult(
...
+  def fee: Bitcoins
+  def modifiedfee: Bitcoins
+  def time: UInt32
+  def height: Int
+  def descendantcount: Int
+  def descendantsize: Int
+  def descendantfees: Bitcoins // Should be BitcoinFeeUnit
+  def ancestorcount: Int
+  def ancestorsize: Int
+  def ancestorfees: Bitcoins // Should be BitcoinFeeUnit
```
Comment (nkohen) [inline bitcoind-rpc/src/main/scala/org/bitcoins/rpc/jsonmodels/BlockchainResult.scala]: ping

**Review by nkohen — COMMENTED** [review]
Looking good :) just a couple nits

File: bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/RawTransactionRpc.scala [hunk 4]
```diff
@@ -88,12 +89,23 @@ trait RawTransactionRpc { self: Client =>
     bitcoindCall[Transaction]("getrawtransaction", params)
   }
 
+  /**
+    * @param maxfeerate On versions before 0.19 set to 0 if you want to enable allowhighfees
```
Comment (nkohen) [inline bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/RawTransactionRpc.scala]: Even for v0.19 `0` means enable `allowhighfees` right? I think just get rid of the first part here

**Review by nkohen — APPROVED** [review]
Just one thing

File: bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/RawTransactionRpc.scala (hunk 4 above)
Comment (benthecarman) [reply bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/RawTransactionRpc.scala]: You're right, that makes more sense

**Pull request merged**